src/Sulu/Bundle/AdminBundle/Resources/js/components/FileUploadButton/FileUploadButton.js

```diff
@@ -33,7 +33,7 @@ export default class FileUploadButton extends React.Component<Props> {
 
         return (
             <Dropzone
-                accept={accept}
+                accept={accept ? {[accept]: []} : undefined}
```

Comment:
we need to bridge this to the new format: https://github.com/react-dropzone/react-dropzone/releases/tag/v13.0.0